rest-api-spec/src/main/resources/rest-api-spec/api/search.json

```diff
@@ -76,6 +76,10 @@
           "type" : "string",
           "description" : "Specify the node or shard the operation should be performed on (default: random)"
         },
+        "prefixed_name": {
+          "type" : "boolean",
+          "description" : "Specify whether aggregation names should be prefixed by their respective types in the response"
+        },
```

Comment:
should also multi_search accept the param? also the corresponding template apis?